src/groups/group.entity.ts

```diff
@@ -43,11 +43,12 @@ export class Group {
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   private constructor() {}
 
-  public static create(name: string, displayName: string): Group {
+  public static create(name: string, displayName: string): Omit<Group, 'id'> {
```

Comment:
I think `special` should be added here because it is a mandatory attribute. Maybe it should have the default value false.
```suggestion
  public static create(name: string, displayName: string, special: boolean): Omit<Group, 'id'> {
```

Reply:
I think this change does not make any sense what so ever. There are only two group that are special and they will be created on startup, so I think we don't need this.

Reply:
It *technically* is a required attribute, but it will only be used to create our two special groups. So I think it is reasonable to not include it here, as the parameter would be set to `false` most of the time.

Reply:
I think we should be consistent here and as you say it is called, why omit it? with the default value, you don't have to specify it.

Reply:
After more discussion with @InnayTool, we came to the conclusion that including `special` here is preferable, as the `create` function will only be used to create the two hard-coded groups and to handle API requests at one or two places.